DataAccessLayer/ServiceInterfaces/IMaterialDataAccess.cs

```diff
@@ -0,0 +1,22 @@
+﻿using ProductionDataAccessLayer.Classes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProductionDataAccessLayer.ServiceInterfaces
+{
+    public interface IMaterialDataAccess
+    {
+        /// <summary>
+        /// Creates a new Material record in the system with the provided details.
+        /// </summary>
+        /// <param name="name">Material's name</param>
+        /// <param name="quantity">Material's quantity</param>
+        /// <returns></returns>
```

Comment:
Here is missing the returns comment.
As we are doing very explicit data accesses and functions it could express itself but I would like to keep the consitency